src/DefaultBuilder/src/ConfigureWebHostBuilder.cs

```diff
@@ -173,7 +173,7 @@ IWebHostBuilder ISupportsStartup.Configure(Action<WebHostBuilderContext, IApplic
         throw new NotSupportedException("Configure() is not supported by WebApplicationBuilder.WebHost. Use the WebApplication returned by WebApplicationBuilder.Build() instead.");
     }
 
-    IWebHostBuilder ISupportsStartup.UseStartup(Type startupType)
+    IWebHostBuilder ISupportsStartup.UseStartup([DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicConstructors | DynamicallyAccessedMemberTypes.PublicMethods)] Type startupType)
```

Comment:
Make this a constant?

Reply:
I'd go the other way and just delete `StartupLinkerOptions`. This is simple enough I like just seeing the flags defined inline.